### ISSUE TITLE
Costing memory 20 times than plain JS

### ISSUE THREAD

**Issue by hank-cp** [ag-grid/ag-grid-vue#8]
**I'm submitting a ...**  (check one with "x")
```
[X] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/ag-grid/ag-grid-vue/blob/master/CONTRIBUTING.md#question
```

**Current behavior**
Given two ag-grid demos with same code (50000 rows data), one in _ag-grid-vue_, another in _plain javascript_, the _ag-grid-vue_ version cost 20 times more memory than the _plain javascript_ version. 
![image](https://user-images.githubusercontent.com/808941/31767466-c0e69ee6-b4fd-11e7-9d1b-3a9f3482cbfc.png)
![image](https://user-images.githubusercontent.com/808941/31767469-c2dc1942-b4fd-11e7-8a80-14ef1c1aadf5.png)

**Expected behavior**
We understand Vue.js will cost more memory than plain JS, but it has to be controlled in an accepted level.

**Minimal reproduction of the problem with instructions**
[plain JS version test case](https://github.com/hank-cp/ag-grid-commonjs-example)
[ag-grid-vue version test case](https://github.com/hank-cp/ag-grid-vue-example)

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**

* **ag-grid-vue version:** 13.3.X
* **ag-Grid/ag-Grid-Enterprise version:** 13.3.X
* **VueJS version:** 2.1
* **Browser:** [all]
* **Language:** [all]

**Comment by hank-cp** [comment]
Please review [this PR](https://github.com/ag-grid/ag-grid-vue/pull/9) , I tried to detach api/columnApi object from Vue.js proxy to avoid observation.

**Comment by seanlandsman** [comment]
Hi

Ok, I've investigated this and can see it's because Vue is making all internal data of the grid reactive.

Until I find a solution for this all I can suggest is that you use Object.freeze on your data, or create your data in created and not in mounted (or any of the other hooks)

AG-946 has been raised our side to track this

thanks for reporting this

**Comment by seanlandsman** [comment]
@hank-cp I think your PR fixes this - great stuff. I'll test and confirm but if so happy days

Thanks for you input on this

**Comment by seanlandsman** [comment]
Hi - I can confirm that your patch fixes this - thanks again

This will be available in our next release, currently scheduled for the end of next week

**Comment by JohnMasterson** [comment]
Just to confirm that this was included in version 14.0.0 which is now available.

Regards, 

John
